problem-03.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+
+int main()
+{
+    int number;
+    printf("Enter a number: ");
+    scanf("%d", &number);
+
+    for (int i = 0; i < 10; i++)
+    {
+        printf("\n%d", number);

```

Comment:
Why are you adding newline character before number? I think it should be added after the number.

Reply:
Updated PR according to your comment